src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,427 @@
+        char_projected_rhs_dt_v_psi) noexcept {
+  // gauge_bc_coeff below is hard-coded here to its default value in SpEC
+  constexpr double gauge_bc_coeff = 1.;
+
+  DataVector inertial_radius(get_size(get<0>(inertial_coords)), 0.);
+  for (size_t i = 0; i < VolumeDim; ++i) {
+    inertial_radius += square(inertial_coords.get(i));
+  }
+  inertial_radius = sqrt(inertial_radius);
+
+  // add in gauge BC
+  for (size_t a = 0; a <= VolumeDim; ++a) {
+    for (size_t b = a; b <= VolumeDim; ++b) {
+      for (size_t c = 0; c <= VolumeDim; ++c) {
+        for (size_t d = 0; d <= VolumeDim; ++d) {
+          bc_dt_v_minus->get(a, b) +=
+              0.5 *
+              (incoming_null_one_form.get(a) * projection_Ab.get(c, b) *
+                   outgoing_null_vector.get(d) +
+               incoming_null_one_form.get(b) * projection_Ab.get(c, a) *
+                   outgoing_null_vector.get(d) -
+               0.5 * incoming_null_one_form.get(a) *
+                   outgoing_null_one_form.get(b) * incoming_null_vector.get(c) *
+                   outgoing_null_vector.get(d) -
+               0.5 * incoming_null_one_form.get(b) *
+                   outgoing_null_one_form.get(a) * incoming_null_vector.get(c) *
+                   outgoing_null_vector.get(d) -
+               0.5 * incoming_null_one_form.get(a) *
+                   incoming_null_one_form.get(b) * outgoing_null_vector.get(c) *
+                   outgoing_null_vector.get(d)) *
+              (get(gamma2) - gauge_bc_coeff * (1. / inertial_radius)) *
+              char_projected_rhs_dt_v_psi.get(c, d);
+        }
+      }
+    }
+  }
+}
+
+template <size_t VolumeDim, typename DataType>
+void add_constraint_dependent_terms_to_dt_v_minus(
+    const gsl::not_null<tnsr::aa<DataType, VolumeDim, Frame::Inertial>*>
+        bc_dt_v_minus,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>& outgoing_null_one_form,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& incoming_null_vector,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& outgoing_null_vector,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>& projection_ab,
+    const tnsr::Ab<DataType, VolumeDim, Frame::Inertial>& projection_Ab,
+    const tnsr::AA<DataType, VolumeDim, Frame::Inertial>& projection_AB,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_plus,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_minus,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>&
+        char_projected_rhs_dt_v_minus,
+    const std::array<DataType, 4>& char_speeds) noexcept {
+  const double mu = 0.;  // hard-coded value from SpEC Bbh input file Mu = 0
+
+  for (size_t a = 0; a <= VolumeDim; ++a) {
+    for (size_t b = a; b <= VolumeDim; ++b) {
+      for (size_t c = 0; c <= VolumeDim; ++c) {
+        for (size_t d = 0; d <= VolumeDim; ++d) {
+          bc_dt_v_minus->get(a, b) +=
```

Comment:
Its indeed Eq. 64 of gr-qc/0512093. The reason it looked different is because this function was assuming that the user-input null vectors / one forms are `t^a (t_a) \pm n^a (n_a)` without the normalization by `1/sqrt(2)`. That is the convention spec uses in its BC code, but I realize it isnt what we want in spectre (e.g. in [`src/PointwiseFunctions/GeneralRelativity/InterfaceNullNormals.hpp`](https://spectre-code.org/group__GeneralRelativityGroup.html#ga88dc523e0343b4f3f01769d2ce5d8421). I have fixed both the documentation and the code to expect the normalizing factor of `1/sqrt(2)` in input null vectors and one forms.

One easy way of verifying that this change is correct is to look at the changes made to the unit test that compares against spec's output. The only  code change needed in that test was to divide the null vectors and one forms by `sqrt(2)` and the test passes, as it should.